src/model/user.rs

```diff
@@ -493,6 +495,9 @@ pub struct User {
     pub name: String,
     /// the public flags on a user's account
     pub public_flags: Option<UserPublicFlags>,
+    /// Optional banner hash.
+    pub banner: Option<String>,
+    pub accent_color: Option<Colour>,
```

Comment:
This will fail to compile if the library is compiled without `utils` support.
```suggestion
    #[cfg(feature = "utils")]
    pub accent_color: Option<Colour>,
    #[cfg(not(feature = "utils"))]
    pub accent_color: Option<u32>,
```

Reply:
Also, might want to change `color` to `colour` and use `#[serde(rename = "...")]` on the field. We prefer British English spelling.

Reply:
> This will fail to compile if the library is compiled without `utils` support.
> ```suggestion
>     #[cfg(feature = "utils")]
>     pub accent_color: Option<Colour>,
>     #[cfg(not(feature = "utils"))]
>     pub accent_color: Option<u32>,
> ```

Maybe that it's time to put `Colour` outside of the util module?

Reply:
I've been considering truncating some of the features we have, `utils` being one. If you look at the contents of the module, the impact it has on compile time is inconsequential, because it's very small and does not depend on any dependency other than `serde`/`serde_json`. I don't any benefit in putting its code behind a feature, and `model` depends on `utils` in a lot of places anyway.

That is a plan for a major release, however, since removal of a feature would be a breaking change.